src/components/avatar/utils.ts

```diff
@@ -63,3 +77,51 @@ export function rgbToHue(rgb: RGB): number {
 export function hexToHue(hex: string): number {
   return rgbToHue(hexToRGB(hex));
 }
+
+/**
+ * For RGB color, find luminance
+ *
+ * @param colorObject { r: number; g: number; b: number }
+ * @param colorObject.r
+ * @param colorObject.g
+ * @param colorObject.b
+ */
+export function rgbToLuminance(colorObject: { r: number; g: number; b: number }): number {
```

Comment:
I believe the `color` module (used by `calcite-color-picker`) already has utilities to get a color's luminance and contrast. Can you look at using those instead?

Reply:
Sure this is my first pass figuring out the basic logic. Still working it out. Will take that note into account.